docs/source/azure/policy/resources/sqlserver.rst

```diff
@@ -11,6 +11,12 @@ Filters
     - Metric Filter - Filter on metrics from Azure Monitor - (see `SQL Server Supported Metrics <https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitoring-supported-metrics#microsoftsqlservers/>`_)
     - Tag Filter - Filter on tag presence and/or values
     - Marked-For-Op Filter - Filter on tag that indicates a scheduled operation for a resource
+- ``firewall-rules`` Filter based on firewall rules. Rules can be specified as x.x.x.x-y.y.y.y or x.x.x.x or x.x.x.x/y.
+  - `include`: the list of IP ranges or CIDR that firewall rules must include.
+  - `equal`: the list of IP ranges or CIDR that firewall rules must match exactly.
+
```

Comment:
We might need to add a note here clarifying the behavior of include.  It isn't clear from this if include is ensuring the exact rules specified exist in the firewall rules, or if it is checking if all IP space listed in include is allowed (potentially across multiple rules, or as a subset of a rule).